src/didactUriCompletionItemProvider.ts

```diff
@@ -302,4 +304,20 @@ export class DidactUriCompletionItemProvider implements vscode.CompletionItemPro
 		}
 		return null;
 	}
+
+	insertDidactBadgeAdoc() : vscode.CompletionItem {
+		const labelText = "Insert Didact Badge";
+		const snippetText = `image:https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/icon/poweredByDidact.png[Powered by Didact,link="https://marketplace.visualstudio.com/items?itemName=redhat.vscode-didact"]`;
```

Comment:
What about pointing to the github repository given that users can use either VS Codium with open-vsx marketplace or Eclipse Che or Eclipse CodeReady Workspaces?

Reply:
Will explore looking at an extension-relative image